packages/twenty-server/src/engine/metadata-modules/flat-object-metadata/validators/validate-flat-object-metadata-identifiers.validator.ts

```diff
@@ -0,0 +1,72 @@
+import { t } from '@lingui/core/macro';
+import {
+  isDefined,
+  isLabelIdentifierFieldMetadataTypes,
+} from 'twenty-shared/utils';
+
+import { type FlatObjectMetadata } from 'src/engine/metadata-modules/flat-object-metadata/types/flat-object-metadata.type';
+import {
+  ObjectMetadataException,
+  ObjectMetadataExceptionCode,
+} from 'src/engine/metadata-modules/object-metadata/object-metadata.exception';
+
+export const validateFlatObjectMetadataIdentifiers = (
```

Comment:
by project convention, I would expect a .validator to be a class and to be named same as the file.

Here it seems to be an util (in this case I would created an util folder below validators)